templates/vhost/vhost_header.erb

```diff
@@ -94,6 +94,16 @@ server {
   passenger_set_cgi_param  <%= key %> <%= @passenger_cgi_param[key] %>;
   <%- end -%>
 <% end -%>
+<% if @passenger_header -%>
```

Comment:
Please call this parameter `passenger_set_header` so it matches the nginx directive name.